packages/governance/src/contractGovernor.js

```diff
@@ -185,6 +191,15 @@ const start = async (zcf, privateArgs) => {
       getUpdatedPoserFacet,
     );
 
+  const { voteOnFilter, createdQuestion: createdFilterQuestion } =
```

Comment:
since this a return value of `setupFilterGovernance` why not have it return with key `createdFilterQuestion`?

I see there's a `FilterGovernor` type that would need to be updated. It's only used here but I see it's made for symmetry with `ApiGovernor` which is returned by two functions.